### DISCUSSION TITLE
scipy completions

### DISCUSSION THREAD

**Discussion by danielpopescu** [microsoft/pyright#6516] [General]
I'm using Pyright as my lsp server with emacs.

Have latest pyright (1.1.336) and use latest stubs from here: https://github.com/microsoft/python-type-stubs (via python.analysis.stubPath) to supplement typeshed.   
For scipy, I get: Pyright [reportMissingTypeStubs]: Stub file not found for "scipy"
It does have stubs for sklearn, so I know it is using stubs from https://github.com/microsoft/python-type-stubs. Any ideas why not using/seeing scipy?

Thank you.

**Comment by erictraut** [comment]
If you haven't already done so, please take a look at [this documentation](https://microsoft.github.io/pyright/#/import-resolution?id=debugging-import-resolution-problems) and follow the suggested troubleshooting steps.
